numpy/lib/npyio.py

```diff
@@ -178,6 +188,7 @@ class NpzFile(Mapping):
     # Make __exit__ safe if zipfile_factory raises an exception
     zip = None
     fid = None
+    MAX_REPR_ARRAY_COUNT = 5
```

Comment:
I would lean towards not making this public, I don't have an opinion about giving it an underscore.  If someone has more files, they can look at `.files`.

Reply:
I tend to agree here - I'd vote for keeping it as simple as possible and not exposing things to the user. As @seberg notes there's already a straightforward way to get the full file listing.

Reply:
Yeah, that makes sense. Made private in [3f4b84d](https://github.com/numpy/numpy/pull/23357/commits/3f4b84d24e6c69371a9582326dd2238200a27b52)